lib/pip.js

```diff
@@ -120,7 +120,7 @@ function installRequirements(
     cmdOptions.push(...pipCmd);
 
     // If enabled slimming, strip out the caches, tests and dist-infos
-    if (options.slim === true) {
+    if (options.slim) {
```

Comment:
When running tests noticed that setting the `slim` parameter via cli as `--slim=true` results in it treated as string.
From this `(options.slim === true)` was evaluated as false every time and the commands were not executed 😅

Reply:
I'm not sure this is what you want either then because Boolean("false") will evaluate to true. I think it should be `options.slim === true || options.slim === "true"`

Reply:
I did the same thing as you for package individually, but David is right, I'd prefer that you use `options.slim === true || options.slim === "true"` and I'll update my use of the same as well.

Reply:
Oh yeah! thanks, that's a really good one, commit on the way 👍